solr/core/src/java/org/apache/solr/cli/AuthTool.java

```diff
@@ -59,6 +60,26 @@ public String getName() {
     return "auth";
   }
 
+  @Override
+  public String getUsage() {
```

Comment:
@janhoy would you mind looking at the output of the help for this tool?  `bin/solr auth -h`?  Did I implement getUsage and getHeader the way you thought they would be?

Reply:
Looks to be as intended, yea. DId not compile the PR to print the help though.